recipes/igwn-lldd-common/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "igwn-lldd-common" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/lscsoft/source/igwn-lldd-common-0.1.0.tar.gz
+  sha256: fa70188e6185e807c92fd10f2b82bcd6046435148aebda18fb3a0f8b419a6ef9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv
+  run_exports:
+    - {{ pin_subpackage("igwn-lldd-common", max_pin="x") }}
```

Comment:
I've never seen a pure python package provide `run_exports`, I'm not sure this is needed here.